elastic/tox.ini

```diff
@@ -5,7 +5,7 @@ skip_missing_interpreters = true
 envlist =
     py{27,37}-{dd}-{0.90}
     py{27,37}-{hub}-{1,2}
-    py{27,37}-{5.4,5.5,5.6,6.0,6.1,6.2,6.3,6.4,7.2}
+    py{27,37}-{5.6,6.3,7.2}
```

Comment:
All the removed versions were using the same code, let's not support so many different versions.